### PR TITLE
Add advanced settings instructions

### PR DESCRIPTION
**Pull request by haydenlam**
### Description

What did you change and why?
 
Added docs for advanced OIDC configuration settings. 
https://konghq.aha.io/epics/KP-E-313

### Testing instructions

Advanced settings are visible here: https://cloud.konghq.com/global/organization/idp-settings#authentication-scheme
Netlify link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/konnect/org-management/oidc-idp.md [hunk 1]
```diff
@@ -55,3 +55,13 @@ You can test the SSO configuration by navigating to the login URI based on the o
 
 You can now manage your organization's user permissions entirely from the IdP
 application.
+
+## Advanced Settings
+
+You can configure customize IdP specific behaviors in the **Advanced Settings** of the OIDC configuration form. The following options are available:
```
Comment (lena-larionova) [inline app/konnect/org-management/oidc-idp.md:61]:
```suggestion
You can configure custom IdP-specific behaviors in the **Advanced Settings** of the OIDC configuration form. The following options are available:
```

File: app/konnect/org-management/oidc-idp.md [hunk 2]
```diff
@@ -55,3 +55,13 @@ You can test the SSO configuration by navigating to the login URI based on the o
...
+
+1. Scopes - Scopes allow you to specify the list of scopes Konnect will request from the IdP. By default, we request the `openid`, `email` and `profile` scopes. The `openid` scope is required and cannot be removed.
```
Comment (lena-larionova) [inline app/konnect/org-management/oidc-idp.md:63]:
```suggestion
1. **Scopes**: Specify the list of scopes {{site.konnect_short_name}} requests from the IdP. By default, {{site.konnect_short_name}} requests the `openid`, `email`, and `profile` scopes. The `openid` scope is required and cannot be removed.
```

File: app/konnect/org-management/oidc-idp.md [hunk 3]
```diff
@@ -55,3 +55,13 @@ You can test the SSO configuration by navigating to the login URI based on the o
...
+2. Claim Mappings - Claim mappings allows you to customize the mapping of required attributes to different claim in the id_token we receive from the IdP. By default, Konnect requires three attributes: Name, Email and Groups, which are mapped as follows:
```
Comment (lena-larionova) [inline app/konnect/org-management/oidc-idp.md:64]:
```suggestion
2. **Claim Mappings**: Customize the mapping of required attributes to a different claim in the `id_token` {{site.konnect_short_name}} receives from the IdP. By default, {{site.konnect_short_name}} requires three attributes: Name, Email, and Groups. The values in these attributes are mapped as follows:
```

File: app/konnect/org-management/oidc-idp.md [hunk 4]
```diff
@@ -55,3 +55,13 @@ You can test the SSO configuration by navigating to the login URI based on the o
...
+    - Name from `name` - used as the Konnect accounts `full_name`.
```
Comment (lena-larionova) [inline app/konnect/org-management/oidc-idp.md:65]:
```suggestion
    - `name`: Used as the {{site.konnect_short_name}} account's `full_name`.
```
I'd say that repeating the value is redundant here.

File: app/konnect/org-management/oidc-idp.md [hunk 5]
```diff
@@ -55,3 +55,13 @@ You can test the SSO configuration by navigating to the login URI based on the o
...
+    - Email from `email` - used as the Konnect account's `email`.
```
Comment (lena-larionova) [inline app/konnect/org-management/oidc-idp.md:66]:
```suggestion
    - `email`: Used as the {{site.konnect_short_name}} account's `email`.
```

File: app/konnect/org-management/oidc-idp.md [hunk 6]
```diff
@@ -55,3 +55,13 @@ You can test the SSO configuration by navigating to the login URI based on the o
...
+    - Groups from `groups` - used to map users to teams defined in the team mappings upon login.
```
Comment (lena-larionova) [inline app/konnect/org-management/oidc-idp.md:67]:
```suggestion
    - `groups`: Used to map users to teams defined in the team mappings upon login.
```

**Review by lena-larionova — CHANGES_REQUESTED** [review]

**Comment by haydenlam** [comment]
Thanks @lena-larionova for the review and suggestions!

**Review by lena-larionova — APPROVED** [review]
Looks pretty good to me! 
https://deploy-preview-5752--kongdocs.netlify.app/konnect/org-management/oidc-idp/#advanced-settings

Feel free to merge whenever you need to.

**Comment by acgoldsmith** [comment]
@haydenlam , This is part of DOCU-3061, so needs to release along with https://github.com/Kong/docs.konghq.com/pull/5376

**Pull request merged**